### PR TITLE
Integrate json-schema-validator container handling

### PR DESCRIPTION
**Pull request by ke4**
Changes:

- [x] Checks whether json-schema-validator image exists locally
  - [x] If not exist pull it from DockerHub
- [x] Check if json-schema-validator container is running
  - [x] If not then start (run) it
- [x] After executing the validation it stops the container
- [x] After stopped the container it prunes it

### CODE REVIEW

File: cli.py [hunk 1]
```diff
@@ -42,8 +51,15 @@ def write_dict(file_path, data_dict):
         '--aap_url', type=str, default='https://api.aai.ebi.ac.uk',
         help='Override the default URL for AAP API.'
     )
+    parser.add_argument(
+        '--log_level', '-l', type=str, default='WARN',
+        help='Override the default logging level.'
+    )
```
Comment (MightyAx) [inline cli.py]: If you add this line:
```
choices=['CRITICAL','ERROR','WARNING','INFO','DEBUG']
```
The user will get more information about their options when running with `--help`,
also the CLI will stop them running the application without the correct parameters.

**Review by MightyAx — APPROVED** [review]
Theres a minor change you could make for usability if you have time, but I'm happy for you to merge.

**Comment by MightyAx** [comment]
Also, can you update the readme to remove the redundant information about docker pull/start/stop?

File: cli.py (hunk 1 above)
Comment (ke4) [reply cli.py]: Agreed, committed and pushed.
Tx

**Pull request merged**